### PR TITLE
feat(range): inherit options from input range

### PR DESCRIPTION
**Pull request by bluelovers**
BREAKING CHANGE: created new range only if not matched input range

### CODE REVIEW

File: classes/range.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ class Range {
     options = parseOptions(options)
 
     if (range instanceof Range) {
+      // inherit options
+      options = { ...{ loose: range.loose, includePrerelease: range.includePrerelease }, ...range.options, ...options }
```
Comment (ljharb) [inline classes/range.js:8]:
```suggestion
      options = { loose: range.loose, includePrerelease: range.includePrerelease, ...range.options, ...options }
```

**Comment by Researchme** [comment]
I have retry my sd card bécasse is entity have delete my application security system professionnalisme but i have the hard drive 2 1the first is at ambassade off usa belgium but is have no information for correctly search i have the the information for find this no choice i m tracking by personal id raison for i have not look the 1.5 million usd this clear is à trop for design me information capital is clôture the bank online is by this way the attack and the project is probable for attack the Grant gov usa in my clouwd i have by chance open one program type enseign and extrait the account bank of this we add this the historique off my site is my case off attack my business the raison off breach security the rapport off WhatsApp is clear all system off i have and account have à breach and now sécurisé darckorcha@hotmail.com and darckorcha38@gmail.com creativeallinbe@gmail.com creativeallinbecompagnie@gmail.com creative.all.in.be@gmail.com creative.all.in.be.compagnie@gmail.com the samedi adresse in Yahoo my adresse actual is ook at Yahoo and find the account miranasantos@gmail.com miranasantos789@gmail.com samedi for Yahoo this the account off initial contact anf this the account off implanté this programe in my data base at verre long time the % off i réalisé this id verre low and off recolt this information 00 but i have work for the police and thanks at revolut and BNP is the alone contact possible for me and i have sending all information hes have contacted nobody Just close my account whitout explication privacypolicyupdates@automattic.com and contact this and demand for confirm my réception i m sas cedric i Liverpool in belgium 158 rue brognier 1070 anderlecht +32493932655 +491765695 Nissan 77040609908 proprietaire administrateur off creative all in compagnie Last information look at www.comunity-freedom-colectif.com its my site is destruct by deletation off data base but hes have 1100 probable a chance for identify look off WhatsApp installation is in the site is the system usine for connection in the web

**Comment by Researchme** [comment]
But is the 1.5 million exist i whish find this and i offert 30 % in github after dépend off the justice i contact maybe iff the amount dommage permet this my first project is à little similar off github à site for help and find the user hacked i have Liver the hell 5 mounth it s an obligation no user have à look this situation thanks all look off the réal or not the 1.5 million i m not understand the codage and programmation thanks for all

**Comment by Researchme** [comment]
Contact this please 
[dpa_11234_75832808413.pdf](https://github.com/npm/node-semver/files/9366233/dpa_11234_75832808413.pdf)

**Comment by wraithgar** [comment]
We're being extra judicious in accepting breaking changes at this point.  I am going to close this due to age and lack of any other issues showing that this is something absolutely needed.  If the community feels this is important we can revisit it.

**Pull request closed without merging**